alf/utils/value_ops.py

```diff
@@ -18,19 +18,26 @@
 from tf_agents.trajectories.time_step import StepType
 from tf_agents.utils import common as tfa_common
 
-from alf.algorithms.rl_algorithm import TrainingInfo
 from alf.utils import common
 
 
-def action_importance_ratio(training_info: TrainingInfo, action_spec,
-                            name_scope, importance_ratio_clipping,
-                            log_prob_clipping, check_numerics,
-                            debug_summaries):
+def action_importance_ratio(action_distribution, collect_action_distribution,
+                            action, action_spec, calling_class_name, scope,
+                            importance_ratio_clipping, log_prob_clipping,
+                            check_numerics, debug_summaries):
     """ ratio for importance sampling, used in PPO loss and vtrace loss.
+
+        Caller has to set tf.name_scope() and nest the call to this function
+            inside the with statement.
+
         Args:
-            training_info: passed in directly from loss classes.
+            action_distribution: Probability of actions under target policy.
+            collect_action_distribution: Probability of action from behavior
+                policy, used to sample actions for the rollout.
+            action: action taken during rollout.
             action_spec (nested BoundedTensorSpec): representing the actions.
-            name_scope: the calling class, 'PPOLoss', 'VTraceLoss' etc..
+            calling_class_name: to decide which clipping method to use.
```

Comment:
classing_class_name is not a good name here, because it's conceivable it's called by other classes. It's better to name as "clipping_mode" and explain the kind of clipping it does.